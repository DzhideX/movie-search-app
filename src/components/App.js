import React from 'react';
import Movies from './MovieSearch';
import Navigation from './Navigation';

function App() {
  return (
    <div data-test='app' className='app'>
        <Navigation />
        <Movies />
    </div>
  );
}

export default App;
