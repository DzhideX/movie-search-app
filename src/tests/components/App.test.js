import { shallow } from 'enzyme';
import App from '../../components/App';
import React from 'react';

test('app renders with no errors',()=>{
    const wrapper = shallow(<App />);   
    expect(wrapper.debug()).toMatchSnapshot();
});