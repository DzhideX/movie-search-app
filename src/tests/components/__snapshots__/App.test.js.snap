// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders with no errors 1`] = `
"<div className=\\"app\\">
  <Connect(Navigation) />
  <Connect(Movies) />
</div>"
`;
